src/main/java/exchange/core2/core/common/cmd/CommandResultCode.java

```diff
@@ -61,6 +61,9 @@
     USER_MGMT_USER_NOT_SUSPENDABLE_NON_EMPTY_ACCOUNTS(-4131),
     USER_MGMT_USER_NOT_SUSPENDED(-4132),
     USER_MGMT_USER_ALREADY_SUSPENDED(-4133),
+    USER_MGMT_USER_NOT_REMOVABLE_HAS_POSITIONS(-4134),
+    USER_MGMT_USER_NOT_REMOVABLE_NON_EMPTY_ACCOUNTS(-4135),
+    USER_SUSPENDED(-4136),
```

Comment:
You can rename to RISK_USER_SUSPENDED(-2005), because this is directly related to the Risk Control section.
This is was a bug indeed, suspended users can not place any new order. Thanks for fixing it.

Reply:
Done. You are most welcome, thanks to you for sharing this awesome work !